docs/basic/install.rst

```diff
@@ -22,9 +22,9 @@ Supported systems
 
 The Psycopg version documented here has *official and tested* support for:
 
-- Python: from version 3.9 to 3.13
+- Python: from version 3.10 to 3.13
 
-  - Python 3.8 supported before Psycopg 3.3
+  - Python 3.8 and 3.9 supported before Psycopg 3.3
```

Comment:
PyPy 3.9 is being dropped too. However I have just tested support to PyPy 3.11 so I am making another change to these lines, which will cause a conflict. I will deal with it myself.